packages/malloy/src/malloy.ts

```diff
@@ -1409,6 +1414,34 @@ export class Explore extends Entity {
   public get structDef(): StructDef {
     return this._structDef;
   }
+
+  public toJson(): SerializedExplore {
+    return {
+      _structDef: this._structDef,
+      sourceExplore: this.sourceExplore?.toJson(),
+      _parentExplore: this._parentExplore?.toJson(),
+    };
+  }
+
+  public toJsonString(): string {
+    return JSON.stringify(this.toJson());
+  }
+
+  public static fromJsonString(json_explore: string): Explore {
+    return Explore.fromJson(JSON.parse(json_explore) as SerializedExplore);
+  }
```

Comment:
Similarly, these don't have counterparts elsewhere, and I'm not sure they're really necessary.

Reply:
Removed the fromJsonString () and toJsonString()`stringify()/parse()`.